tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/country_language_population_raw.txt

```diff
@@ -2,175 +2,175 @@
 #######
 #WARNING: this file is in TSV format (for pasting from a spreadsheet); make sure you maintain tabs!
 #######
-Afghanistan	AF	"34,940,837"	28%	"69,450,000,000"	official_regional	Baluchi	bal	"233,000"			
-Afghanistan	AF	"34,940,837"	28%	"69,450,000,000"		Hazaragi	haz	5.9%			
+Afghanistan	AF	"34,940,837"	28%	"69,450,000,000"	official_regional	Baluchi	bal	"233,000"
```

Comment:
Hmm, I do not think the trailing tabs should have been deleted, this was set up for import to a spreadsheet.

Reply:
would trailing tabs affect the spreadsheet though?

Reply:
Perhaps not, though I guess it could depend on the spreadsheet.